community/mei-by-laws.md

```diff
@@ -73,41 +77,51 @@ The Board is self-organizing; that is, there will be no formal regulations on ho
+
+### 5.3 Terms & Elections
 
-Board members may resign from their commitments at any time but are expected to give adequate notice. After accepting the resignation, the Board may decide on how to proceed concerning the newly vacant position. Options should include but are not limited to holding a special election, appointing a replacement, or leaving the position vacant.
+The elected members of the Board each are elected for a three year term. Terms on the Board are staggered so that three positions are open for election each year. Terms of office begin January 1 at 2 pm UTC.
 
-In case of a special election to fill a vacancy between regular elections the newly elected Board member will serve for the remainder of the term of the vacant position.
+Exception to annual elections and term lengths will be made for the initial election of the Board and in case a Board member resigns.
 
 #### 5.3.1 Eligibility
 
 In order to be available for election to the Board, candidates must hold Personal Membership in MEI. Individuals who have previously served on the Board will be eligible for re-appointment.
 
 #### 5.3.2 Election procedure
 
-In order to determine a range of candidates, a four-week nomination phase will precede the elections, in which self-nominations are permitted. Individuals will be informed of their nomination and asked to confirm their willingness to serve on the Board determined. Elections will be held in a timely manner after determination of a final list of candidates. A ranked voting system without minimum voter turnout will be applied.
+In order to determine a range of candidates, a two-week nomination phase will precede the elections, in which self-nominations are permitted. Individuals will be informed of their nomination and asked to confirm their willingness to serve on the Board determined. Elections will be held in a timely manner after determination of a final list of candidates. Elections should be open for at least two weeks. Elections should close before the start of the terms of office of the elected candidates.
+A ranked voting system without minimum voter turnout will be applied.
 
 Each member holding Personal Membership in MEI has the right to vote in Board member elections.
 
+### 5.4 Resignation from the Board
+
+Any member of the Board may resign from the Board and their corresponding commitments at any time on written notification (physical or electronic) to the administrative chair or the whole Board.
+
+After accepting the resignation, the Board may decide on how to proceed concerning the newly vacant position. Options should include but are not limited to holding a special election, appointing a replacement, or leaving the position vacant.
```

Comment:
"should" is an appropriate word here, because this sentence really is a suggestion, not a command.

Reply:
Thank you!